crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -59,13 +59,22 @@ type KeyDiagnosticFields = (
     Severity,
 );
 
-static EXPECTED_TOMLLIB_DIAGNOSTICS: &[KeyDiagnosticFields] = &[(
-    DiagnosticId::lint("unused-ignore-comment"),
-    Some("/src/tomllib/_parser.py"),
-    Some(22299..22333),
-    "Unused blanket `type: ignore` directive",
-    Severity::Warning,
-)];
+static EXPECTED_TOMLLIB_DIAGNOSTICS: &[KeyDiagnosticFields] = &[
+    (
+        DiagnosticId::lint("no-matching-overload"),
+        Some("/src/tomllib/_parser.py"),
+        Some(2329..2358),
+        "No overload of bound method `__init__` matches arguments",
+        Severity::Error,
+    ),
```

Comment:
Have you looked at the cause of this? We were so close to zero tomllib false positives 😆

Reply:
This is https://github.com/astral-sh/ruff/issues/16994. The diagnostic is on the `Output` reference [here](https://github.com/astral-sh/ruff/blob/4bcf1778fa0a223653c75ef44695428ae309430f/crates/ruff_benchmark/resources/tomllib/_parser.py#L76).  `Output` [inherits from `NamedTuple`](https://github.com/astral-sh/ruff/blob/4bcf1778fa0a223653c75ef44695428ae309430f/crates/ruff_benchmark/resources/tomllib/_parser.py#L227), which is the cause of a lot of the new ecosystem false positives, too.